src/infrastructure/notification/notifications/created-key-result-comment.notification.ts

```diff
@@ -124,10 +136,12 @@ export class CreatedKeyResultCommentNotification extends BaseNotification<
     const keyResult = await this.core.dispatchCommand<KeyResultInterface>('get-key-result', {
       id: keyResultID,
     })
+    const cycle = await this.getCycle(keyResult)
     const owner = await this.core.dispatchCommand<UserInterface>('get-key-result-owner', keyResult)
```

Comment:
Seria possivel fazer isso? Assim diminuimos o tempo aguardando por IO.

```suggestion
    const [ cycle, owner ] = await Promise.all([
      this.getCycle(keyResult),
      this.core.dispatchCommand<UserInterface>('get-key-result-owner', keyResult),
    ])
```